landlab/bmi/bmi_bridge.py

```diff
@@ -243,17 +253,40 @@ def __init__(self):
             self._clock = None
             super(BmiWrapper, self).__init__()
 
+            self._input_var_names = tuple(
+                set(self._cls._input_var_names) | {"boundary_condition_flag"}
+            )
+            # self._cls._input_var_names = tuple(
+            #     set(self._cls._input_var_names) | {"boundary_condition_flag"}
+            # )
+            self._output_var_names = tuple(
+                set(self._cls._output_var_names) | {"boundary_condition_flag"}
+            )
+            # self._cls._output_var_names = tuple(
+            #     set(self._cls._output_var_names) | {"boundary_condition_flag"}
+            # )
+            self._var_mapping = self._cls._var_mapping.copy()
+            self._var_units = self._cls._var_units.copy()
+            self._var_doc = self._cls._var_doc.copy()
+
+            self._var_mapping["boundary_condition_flag"] = "node"
+            self._var_units["boundary_condition_flag"] = ""
+            self._var_doc["boundary_condition_flag"] = "boundary condition flag of grid nodes"
+            # self._cls._var_mapping["boundary_condition_flag"] = "node"
```

Comment:
- Recommend remove comments
- Are our boundary condition flags at all universal? (my assumption is no...) is there a more universal format that makes sense to use? I imagine this won't change between now and (tomorrow/the asap time when this is merged), but is it worth making an issue to keep track of doing this somewhat correctly?

Reply:
I don't think our boundary condition flags are at all universal but I'm not sure that anyone's are.